Extensions/Physics2Behavior/JsExtension.js

```diff
@@ -758,7 +758,7 @@ module.exports = {
 
     aut
       .addAction(
-        'SetSleepingaAllowed',
+        'SetSleepingAllowed',
```

Comment:
This was a typo indeed but was not a bug as this was invisible for users. 
Changing the name means the old actions "SetSleepingaAllowed" won't work anymore though, so we'll have to use `addDuplicatedAction`:

`aut.addDuplicatedAction("SetSleepingaAllowed", "SetSleepingAllowed").SetHidden();`

so that internally the old action is still existing and works for old games :)

Reply:
thanks!